sympy/polys/domains/algebraicfield.py

```diff
@@ -351,25 +351,18 @@ def to_alg_num(self, a):
     def to_sympy(self, a):
         """Convert ``a`` of ``dtype`` to a SymPy object. """
         # Precompute a converter to be reused:
-        if not hasattr(self, '_converter'):
-            self._converter = _make_converter(self)
-
-        return self._converter(a)
+        conv = getattr(self, '_converter', None)
+        if conv is None:
+            conv = self._converter = _make_converter(self)
+        return conv(a)
```

Comment:
This just looks like a rephrasing. Is it more efficient?

Reply:
The `hasattr` call does an attribute lookup and so does `self._converter`. With `getattr` there is only one loop up. This is a very minor point and likely not significant compared to the cost of actually calling `conv` but in general there is a computational cost in an attribute lookup.

Reply:
Sounds good.